pandas/tests/_testing/test_warnings.py

```diff
@@ -0,0 +1,138 @@
+""""
+Test module for testing ``pandas._testing.assert_produces_warning``.
+"""
```

Comment:
There is already a file with tests for this, in `tests/util/test_assert_produces_warning.py`, can you move this there?

Reply:
Oh, I completely missed it. I moved tests as requested.